.travis.yml

```diff
@@ -9,6 +9,6 @@ jdk:
 before_install:
   - sudo apt-get install jq
   - wget -O ~/codacy-coverage-reporter-assembly-latest.jar $(curl https://api.github.com/repos/codacy/codacy-coverage-reporter/releases/latest | jq -r .assets[0].browser_download_url)
-install: mvn install -DskipTests -Dgpg.skip
```

Comment:
this should not be removed. the `install` just compiles the library. then the `script` section below runs the tests

see https://docs.travis-ci.com/user/languages/java/#Projects-Using-Maven